### DISCUSSION TITLE
Downloading preset dashboards

### DISCUSSION THREAD

**Discussion by fbo84** [Dynatrace/dynatrace-configuration-as-code#451] [Q&A]
Hi all
I'm using monaco to backup my dashboards, but I have recently discovered that it does download the dashboards with "preset: true" attribute. This is annoying me, because as an admin most of them are preset in order to publish them to all users.
Is there a way to avoid this behavior ?
Thanks
Franck

**Comment by ChaymaeGit** [comment]
Hello , I see your answer on https://golangissues.com/issues/489776. I face the same problem while trying to backup preset dashboards whith Monaco and still don't know how ?Any help please?

Thank you
